src/authorizations/components/redesigned/GenerateTokenDropdown.tsx

```diff
@@ -29,14 +32,17 @@ const GenerateTokenDropdown: FC<ReduxProps & GenerateTokenProps> = ({
 
   const handleAllAccess = () => {
     showOverlay('add-master-token', null, dismissOverlay)
+    event('all access overlay accessed')
```

Comment:
```suggestion
    event('generate_token_dropdown.all_access_overlay.opened')
```

Alex does his own thing with a less structured style of event name (which is why you might see some event names like 'all access overlay accessed'), but Kristina and I would prefer a more organized style, that has a more categorical approach because it's easier to search for, and it provides some kind of organization.

You don't have to use the exact suggestion, but it should follow a general pattern like:

`component.action.outcome`

Reply:
that totally makes sense. I will update the event names accordingly